db/index.js

```diff
@@ -53,8 +53,8 @@ let save = review => {
   })
 }
 
-let find = callback => {
-  Review.find({}).exec( (err, res) => {
+let find = (inputGame, callback) => {
+  Review.find({game: inputGame}).exec( (err, res) => {
```

Comment:
What is exec doing here?

Reply:
I intended to use it to continue the callback pattern

Reply:
Found this: Answered my question.